zerver/webhooks/gitlab/view.py

```diff
@@ -88,6 +89,34 @@ def get_issue_event_body(payload: Dict[str, Any], action: str,
         title=payload['object_attributes'].get('title') if include_title else None
     )
 
+def get_confidential_issue_created_event_body(payload: Dict[str, Any],
+                                              include_title: Optional[bool]=False) -> str:
+    description = payload['object_attributes'].get('description')
+    # Filter out multiline hidden comments
+    if description is not None:
+        description = re.sub('<!--.*?-->', '', description, 0, re.DOTALL)
+        description = description.rstrip()
+
+    return get_confidential_issue_event_message(
```

Comment:
About my comment above, we absolutely do not need `get_confidential_issue_event_message` in `zerver.lib.webhooks.git`, just call `get_pull_request_event_message` (in `zerver/lib/webhooks/git.py`) directly here.